### PR TITLE
Basic docs in README

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+![Build](https://github.com/conan-io/conan-vs-extension/actions/workflows/main.yml/badge.svg?branch=develop2)
+
+Conan, the C and C++ package manager, is now directly integrated into your IDE. List
+available libraries, install, and use them without leaving Visual Studio.
+
+## Using the Extension
+
+> **Note**
+> 
+> For a detailed explanation of how to use the extension, please visit [our documentation
```
Comment (AbrilRBS) [inline README.md:12]: The text says extension, but this just links to the general docs. Maybe a wording change?

**Review by AbrilRBS — APPROVED** [review]
Reads nicely :)

File: README.md (hunk 1 above)
Comment (czoido) [reply README.md:12]: Maybe we can use this link: https://docs.conan.io/2/integrations/visual_studio.html as we have to update these docs to also documento the use of the extension?

Comment (danimtb) [reply README.md:12]: yes, we have to link to a specific section

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+
+Now, if you initiate the build process for your project, the pre-build event will trigger
+Conan to install the packages and inject the necessary properties into the project,
+enabling Visual Studio to compile and link against those libraries.
+
+![Build project](./.github/readme/build-project.png)
+
+> ⚠️ **Warning**: The initial compilation might fail if Visual Studio does not have
+> sufficient time to process the injected properties. If this happens, simply rebuild the
+> project, and it should build successfully.
```
Comment (danimtb) [inline README.md:57]: 😉

**Review by danimtb — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+
+Conan, the C and C++ package manager, is now directly integrated into your IDE. List
+available libraries, install, and use them without leaving Visual Studio.
+
+## Using the Extension
+
+> **Note**
+> 
+> For a detailed explanation of how to use the extension, please visit [our documentation
+> page](https://docs.conan.io/2/).
```
Comment (czoido) [inline README.md]:
```suggestion
> page](https://docs.conan.io/2/integrations/visual_studio.html).
```

**Pull request merged**